config/kubernetes/sig-autoscaling/teams.yaml

```diff
@@ -14,10 +14,15 @@ teams:
   autoscaler-maintainers:
```

Comment:
The autoscaling cleanups are also legit. Ref: https://github.com/kubernetes/org/pull/4343

These can be safely dropped.

Reply:
Note: Changes to the whole file can be dropped. GitHub UI is weird and doesn't show the whole selection.

Reply:
dropped the respective commit. Thanks!